cli/azd/cmd/login.go

```diff
@@ -205,8 +205,8 @@ func countTrue(elms ...bool) int {
 }
 
 func (la *loginAction) login(ctx context.Context) error {
-	if la.flags.clientID != "" || la.flags.tenantID != "" {
-		if la.flags.clientID == "" || la.flags.tenantID == "" {
+	if la.flags.clientID != "" {
```

Comment:
Will need to update the usage for `login` command to reflect that tenant ID can be used optionally.